04-Multi-Band-Raster.Rmd

```diff
@@ -329,18 +386,38 @@ RGB_brick <- brick(RGB_stack)
 
 #view size of the brick
 object.size(RGB_brick)
+```
+
+Since all the bands are stored within the brick object the size is much bigger 
+for a brick than for a stack.  
 
+PlotRGB can still be used for plotting bricks.  
+``` {r plot-brick}
 #plot brick
-plot(RGB_brick,
-     col=grayscale_colors)
+plotRGB(RGB_brick)
 
 ```
 
 
-#Challenge
+#Challenge: Methods for Calling on an Object
```

Comment:
I like this challenge but seems very out of place here.  A good option might be to keep it here for pull request and to start an issue about it.